runtests.py

```diff
@@ -84,7 +84,7 @@
 def run_cmd(name: str) -> int:
     status = 0
     cmd = cmds[name]
-    print('run %s: %s' % (name, cmd))
+    print('run {}: {}'.format(name, cmd))
```

Comment:
Should be an f-string. (From in-person conversation, need to run pyupgrade again because it doesn't have stable output.)